spec/README.mediawiki

```diff
@@ -23,7 +23,8 @@ While they remove the trusted third party (TTP), they insert their own products
 as a means to capture trading fees, which replaces the TTP friction with a
 new platform friction.
 
-A more robust solution is based on atomic swap technology [https://github.com/decred/atomicswap &#91;2&#93;],
+A more robust solution is based on [https://github.com/decred/atomicswap atomic swap technology]
+[[references.mediawiki|<nowiki>[2]</nowiki>]],
```

Comment:
changing unreadable code to `<nowiki></nowiki>` as it makes more sense for humans.